app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,38 @@
+class SessionsController < ApplicationController
+    # Sign in form
+    def new
+    end
+    
+    # Submitting the sign in form
+    # TODO case insensitive email look ups
```

Comment:
@ryfauber I know you're big on case insensitivity—I promise a pull request soon ;)

Reply:
you better!

Reply:
Insensitivity never makes sense.